dogs-initial/src/main/java/com/lambdaschool/dogsinitial/controller/DogController.java

```diff
@@ -36,4 +40,15 @@
                 findDogs(d -> d.getBreed().toUpperCase().equals(breed.toUpperCase()));
         return new ResponseEntity<>(rtnDogs, HttpStatus.OK);
     }
+
+    // localhost:2019/data/dogtable
+    @GetMapping(value = "/dogtable")
+    public ModelAndView displayEmployeeTable()
+    {
```

Comment:
Quick heads-up for your comment on line 44: the actual URI for this endpoint will be `"/dog/dogs/dogtable"` - if you want it to match the URI in your comment, consider these two settings:
• `server.servlet.context-path=/dog`
• The `@RequestMapping("/dogs")` annotation on top of your `DogController` class

Take one of those out and change the path for the other to `"/data"`.